### PR TITLE
Releasing 2.8.0b4

### PR DESCRIPTION
**Pull request by unho**
This replaces #5167

Add release notes and bump version for newer yet-to-be-published 2.8.0b4 release. Having some release notes already in place will minimize the number of commits that need to be checked just before release, making the release process faster.

**Note:** updated up to commit 05e582b6 (included).

### CODE REVIEW

File: docs/releases/2.8.0.rst [hunk 1]
```diff
@@ -15,9 +15,11 @@ If you want to try it, check one of the following:
 Changes in Requirements
 =======================
 
-- Django>=1.9.10,<1.10
+- Django>=1.9.11,<1.10
 - zlib1g-dev required for lxml
 - MySQL support uses mysqlclient instead of MySQLdb
+- django-transaction-hooks is no longer used
```
Comment (dwaynebailey) [inline docs/releases/2.8.0.rst]: These are changes since 2.7.6? So not sure we've ever released with that requirement

File: docs/releases/2.8.0.rst [hunk 2]
```diff
@@ -15,9 +15,11 @@ If you want to try it, check one of the following:
...
+- Updated and pinned PyPI requirements
```
Comment (dwaynebailey) [inline docs/releases/2.8.0.rst]: Not a major issue I'd say, just do that in the main body

File: docs/releases/2.8.0.rst [hunk 3]
```diff
@@ -29,82 +31,162 @@ Major Changes
...
+    - Display original and translations side-by-side for TM results
+    - amaGama Translation Memory is now queried using CORS
+
+  - Perform all highlighting in the client
+  - Clearly present plurals to translator
+  - Allow selecting similar translations' text in editor
+  - Force word wrapping on long strings with no spaces
+  - Improvements on timeline
+  - Cross-language translation is now restricted to admins since translators
+    don't require them and it heavily impacts performance.
```
Comment (dwaynebailey) [inline docs/releases/2.8.0.rst]: I'd reword, it sounds like a very opinionated assumption,  "Cross-language translation is now restricted to admins as they heavily impacts performance and translators are unlikely to require them"

File: docs/releases/2.8.0.rst [hunk 4]
```diff
@@ -29,82 +31,162 @@ Major Changes
...
 
-  - Disabled upload for non-PO projects as conflict handling currently only works in PO
+  - Disabled upload for non-PO projects as conflict handling currently only
+    works in PO
   - Admins can upload translations as other user
 
 - New Machine Translation providers:
 
-  - `Caighdeán <https://github.com/kscanne/caighdean/blob/master/API.md>`_ - Irish
+  - `Caighdeán (Irish) <https://github.com/kscanne/caighdean/blob/master/API.md>`_
```
Comment (dwaynebailey) [inline docs/releases/2.8.0.rst]: The former was better, that was the name of the service

File: docs/releases/2.8.0.rst [hunk 5]
```diff
@@ -29,82 +31,162 @@ Major Changes
...
 - Documentation updates
 
 
 Pootle FS
 ---------
 
-Pootle FS allows to synchronize Pootle against a filesystem, or VCS, allowing
-to handle conflicts and several other situations that previously required
-ad-hoc hacks.
+:ref:`Pootle FS <pootle_fs>` allows to synchronize Pootle against a filesystem,
```
Comment (dwaynebailey) [inline docs/releases/2.8.0.rst]: allows to/enables synchornization of

File: docs/releases/2.8.0.rst [hunk 6]
```diff
@@ -29,82 +31,162 @@ Major Changes
...
+or VCS, allowing to handle conflicts and several other situations that
```
Comment (dwaynebailey) [inline docs/releases/2.8.0.rst]: VCS/version control system

Comment (dwaynebailey) [inline docs/releases/2.8.0.rst]: . Handling conflict resolution and other situations of two files being out of sync

File: docs/releases/2.8.0.rst [hunk 7]
```diff
@@ -140,16 +223,19 @@ Changes in settings
...
   documenation on :setting:`MIN_SIMILARITY <POOTLE_TM_SERVER-MIN_SIMILARITY>`.
 - :setting:`POOTLE_SEARCH_BACKEND` was added, to allow configuring the search
   backend to be used.
 - Changed the default value for `ACCOUNT_SESSION_REMEMBER
   <https://django-allauth.readthedocs.io/en/latest/configuration.html>`_ so now
-  sessions are always remembered.  
+  sessions are always remembered.
+- :setting:`POOTLE_EMAIL_FEEDBACK_ENABLED` was added, to allow disabling
+  sending emails to suggesters when suggestions are accepted or rejected.
+- Removed ``POOTLE_STATS_REFRESH_ATTEMPTS_COUNT`` setting.
```
Comment (dwaynebailey) [inline docs/releases/2.8.0.rst]: Did we ever release with this?

File: pootle/constants.py [hunk 8]
```diff
@@ -6,4 +6,4 @@
 # or later license. See the LICENSE file for a copy of the license and the
 # AUTHORS file for copyright and authorship information.
 
-VERSION = (2, 8, 0, 'beta', 3)
+VERSION = (2, 8, 0, 'beta', 4)
```
Comment (dwaynebailey) [inline pootle/constants.py]: Note for future, we need to change this up after each release as we've been working on beta4 up until now and this just confuses people.

File: docs/releases/2.8.0.rst (hunk 1 above)
Comment (unho) [inline docs/releases/2.8.0.rst]: Yes, this [arrived in 2.7.1](https://github.com/translate/pootle/commit/cd2fc9fbc180fd3dd62805fa03ca4a4ebd9de521).

File: docs/releases/2.8.0.rst (hunk 7 above)
Comment (unho) [inline docs/releases/2.8.0.rst]: No, removed.

**Comment by dwaynebailey** [comment]
lgtm
